WBooks/Library/Cell/LibraryTableViewCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  LibraryTableViewCell.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 12/08/21.
+//
+
+import UIKit
+
+class LibraryTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var containerView: UIView!{
```

Comment:
add space between "! {"
`@IBOutlet weak var containerView: UIView! {`

and

`didSet {`

Reply:
Done